src/components/LmsConfigurations/BlackboardIntegrationConfigForm.jsx

```diff
@@ -0,0 +1,284 @@
+              for="active"
+            >
+              <label htmlFor="active">Active</label>
+              <Input
+                type="checkbox"
+                id="active"
+                name="active"
+                className="ml-3"
+                checked={active}
+                onChange={() => this.setState(prevState => ({ active: !prevState.active }))}
+              />
+            </ValidationFormGroup>
+          </div>
+        </div>
+        <div className="row">
+          <div className="col col-4">
+            <ValidationFormGroup
+              for="blackboardBaseUrl"
+              invalid={invalidFields.blackboardBaseUrl}
+              invalidMessage="Blackboard Instance URL is required."
+              helpText="Your Blackboard instance URL. Make sure to include the protocol (ie https/http)"
+            >
+              <label htmlFor="blackboardBaseUrl">Blackboard Instance URL</label>
+              <Input
+                type="text"
+                id="blackboardBaseUrl"
+                name="blackboardBaseUrl"
+                defaultValue={config ? config.blackboardBaseUrl : null}
+              />
+            </ValidationFormGroup>
+          </div>
+        </div>
+        <div className="row">
+          <div className="col col-4">
+            <ValidationFormGroup
+              for="clientId"
+              invalid={invalidFields.clientId}
+              invalidMessage="Blackboard client ID is required."
+              helpText="This should match the API Client ID found on Blackboard."
+            >
+              <label htmlFor="clientId">Blackboard Client ID</label>
+              <Input
+                type="text"
+                id="clientId"
+                name="clientId"
+                defaultValue={config ? config.clientId : null}
+              />
+            </ValidationFormGroup>
+          </div>
+        </div>
+        <div className="row">
+          <div className="col col-4">
+            <ValidationFormGroup
+              for="clientSecret"
+              invalid={invalidFields.clientSecret}
+              invalidMessage="Blackboard client secret is required."
+              helpText="This should match the API Client secret found on Blackboard."
+            >
+              <label htmlFor="clientSecret">Blackboard Client Secret</label>
+              <Input
+                type="text"
+                id="clientSecret"
+                name="clientSecret"
+                defaultValue={config ? config.clientSecret : null}
+              />
+            </ValidationFormGroup>
+          </div>
+        </div>
+        <div className="row">
+          <div className="col col-4">
+            <ValidationFormGroup
+              for="refreshToken"
+              helpText="The Blackboard API's refresh token token. This should be automatically propagated once you visit the oauth complete endpoint."
+              // invalid={invalidFields.refreshToken}
```

Comment:
make sure to just delete this and the line below it (I'm guessing we don't want to require this field?)

Reply:
lol whoops, good catch!